plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/transport/http/OpenLineageHttpTransport.java

```diff
@@ -57,7 +59,9 @@ public HttpTransport buildTransport()
                         this.tokenProvider,
                         this.urlParams,
                         this.headers,
-                        null,
+                        (this.compression == OpenLineageHttpTransportConfig.Compression.GZIP
+                            ? HttpConfig.Compression.GZIP
+                            : null),
```

Comment:
The indentation is wrong, and the parentheses look redundant. 

We can make it a single line because it's a simple expression.